Carthage-xcfilelist/app-input-files.xcfilelist

```diff
@@ -9,7 +9,6 @@ $(SRCROOT)/Carthage/Build/iOS/OnePasswordExtension.framework
 $(SRCROOT)/Carthage/Build/iOS/Prelude.framework
 $(SRCROOT)/Carthage/Build/iOS/Prelude_UIKit.framework
 $(SRCROOT)/Carthage/Build/iOS/ReactiveExtensions.framework
-$(SRCROOT)/Carthage/Build/iOS/ReactiveExtensions_TestHelpers.framework
```

Comment:
Although this didn't appear to be problematic for the upload, we should not be bundling our test helpers with the app.